frontend/src/pages/Traces/TraceLogs.tsx

```diff
@@ -130,6 +132,7 @@ export const TraceLogs: React.FC = () => {
 								query={query}
 								selectedCursor={undefined}
 								fetchMoreWhenScrolled={fetchMoreWhenScrolled}
+								bodyHeight="400px"
```

Comment:
Can't use a percentage here since the panel is scrollable. We could look into making this panel not scrollable like the sessions' network tab, but will be a larger effort. I believe this is a good compromise/improvement over the previous state